colour/appearance/tests/fixtures/cam16.csv

```diff
@@ -0,0 +1,5 @@
+Case,X,Y,Z,X_w,Y_w,Z_w,L_A,Y_b,F,c,N_c,J,C,h,s,Q,M,H
```

Comment:
Where is the test data from? Should we add a source?

Reply:
Oh this is generated as of now, which is very unfortunate. I sent a mail to M. R. Luo start of January but did not hear back from him. I will maybe check if Stephen Westland can get a hook on him.

Reply:
That's okay I guess, but we should probably document that somewhere.

Reply:
Not that I'm aware of unfortunately. Any particular place you would see a mention about that?